src/pages/jobs.module.scss

```diff
@@ -16,6 +16,14 @@
 	color: var(--waterloo);
 	margin: 0 auto 60px;
 	text-align: center;
+
+	& a {
+		color: var(--jungle-green);
+	}
+
+	& a:hover {
+		color: var(--jungle-green-hover);
+	}
```

Comment:
- parent's `&` scss selector is redundant;
- Since `hover` state is also part of the `a` selector we can define this rule using `&` scss selctor:
```scss
a {
  color: var(--jungle-green);
  
  &:hover {
    color: var(--jungle-green-hover);
  }
}
```